package.json

```diff
@@ -25,41 +25,31 @@
   },
   "devDependencies": {
     "@babel/eslint-parser": "^7.17.0",
-    "@babel/plugin-proposal-class-properties": "^7.10.4",
-    "@babel/plugin-proposal-decorators": "^7.10.5",
-    "@babel/plugin-transform-runtime": "^7.10.5",
-    "@babel/preset-react": "^7.10.4",
     "@bigtest/interactor": "^0.7.2",
     "@bigtest/mocha": "^0.5.1",
     "@bigtest/react": "^0.1.2",
-    "@folio/eslint-config-stripes": "^6.1.0",
-    "@folio/stripes": "^8.0.0",
-    "@folio/stripes-cli": "^2.4.0",
-    "@folio/stripes-core": "^9.0.0",
+    "@folio/eslint-config-stripes": "^7.0.0",
+    "@folio/jest-config-stripes": "^2.0.0",
+    "@folio/stripes": "^9.0.0",
+    "@folio/stripes-cli": "^3.0.0",
+    "@folio/stripes-core": "^10.0.0",
     "@formatjs/cli": "^4.2.20",
-    "@testing-library/jest-dom": "^5.11.1",
-    "@testing-library/react": "^11.0.2",
-    "@testing-library/react-hooks": "^5.1.2",
     "axe-core": "~4.1.4",
-    "babel-jest": "^26.3.0",
-    "babel-polyfill": "^6.26.0",
```

Comment:
`babel-polyfill` is used in `test/bigtest/index.js`

Reply:
Bigtest tests are not used anymore, I created a ticket to remove everything related to it